stacks/apps/dovetail-router.yml

```diff
@@ -947,10 +949,9 @@ Resources:
         const SERVICE_NAME = process.env.ECS_SERVICE_NAME;
         const SCALE_TO = {
           '815/feed-rss.xml': 4,
-          '284/feed-rss.xml': 9,
-          '70/feed-rss.xml': 24,
+          '70/feed-rss.xml': 30,
           'criminal/feed-rss.xml': 16,
-          'themoth/feed-rss.xml': 16,
+          'themoth/feed-rss.xml': 30,
```

Comment:
284 is a test show, so no need to scale on that.  (And especially no need to scale in past the minimum).

TTD/Moth should be scaled out to the max when the private feeds change.